### ISSUE TITLE
Migrate away from third party cookies

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#527]
### What happened?

Chrome is restricting third-party cookies:

https://developers.google.com/privacy-sandbox/3pcd

Google Analytics uses this. Make the necessary changes to stop using them.

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

macOS

**Comment by lpjjj1222** [comment]
@amyjko Hi Amy, just want to confirm the issue needed for change. Do I just nned to disable Google Analytics for Wordplay, since Google Analytics uses third-party cookies, which are blocked by Chrome? Please correct me if I am wrong =)

**Comment by amyjko** [comment]
@lpjjj1222 That is not what this issue is about. Google Analytics will still exist, but they're changing something about how they use cookies to do tracking. We have to understand what changes they are making and then make any changes on our end in response to their changes. The task here is to read the policy change, generate tasks, and implement whatever is required, not to disable Google Analytics.

**Comment by annieli555** [comment]
Hi @amyjko! I'm interested in working on this issue. May I be assigned?

**Comment by amyjko** [comment]
Sure! Assigned. Lots to learn in the Google documentation about what we need to do.

**Comment by annieli555** [comment]
Hi @amyjko! I began following the instructions at https://developers.google.com/privacy-sandbox/3pcd and completed the [Audit cookies](https://developers.google.com/privacy-sandbox/cookies/prepare/audit-cookies) step. I believe we don't use any third-party cookies on the site at the moment. I verified this three ways:

1. The instructions said, as a starting point, to search your code for instances where you set the `SameSite` cookie to `None`. There are no occurrences of `SameSite=None` in the codebase.

2. All the existing cookies I see in Application > Cookies in Chrome DevTools are set from wordplay.dev, so they are first-party cookies:
![Screenshot 2024-11-22 at 8 15 22 PM](https://github.com/user-attachments/assets/bba5b3d7-acfd-43d6-9f0e-087379d1c88d)

3. Google's [Privacy Sandbox Analysis Tool (PSAT)](https://developers.google.com/privacy-sandbox/blog/psat-announcement) shows no third-party cookies on wordplay.dev, only first-party cookies.
![Screenshot 2024-11-22 at 8 25 18 PM](https://github.com/user-attachments/assets/b1dee119-450d-4073-9d79-8456cfe059e6)

I also verified 2 + 3 from several pages like https://wordplay.dev/learn, and in Incognito mode in case any of my browser extensions were interfering with third-party cookies.

Please let me know if this is different from your understanding of how cookies are used on the site. Otherwise, I think there are no further tasks to be done for this issue!

**Comment by amyjko** [comment]
Thank you for investigating! I think your analysis looks complete and correct to me; we use Google Analytics in limited ways, just using 1st party, so all of this sounds correct. Let's consider this issue done.
